### PR TITLE
Add bulgarian translations (BG)

### PR DESCRIPTION
**Pull request by ivcho02**

### CODE REVIEW

**Comment by paulbalandan** [comment]
Hi! Thank you for your PR!

For this to be merged, we need:
1. A test case for the Bulgarian language. `Translations\Tests\BulgarianTranslationTest`
2. Remove `bg` from `MISSING.md`
3. Add `bg` to `README.md`
4. Pass all tests.

**Comment by ivcho02** [comment]
HI @paulbalandan ,
Thanks for the reply, I will take care of this! 

Kind Regards,
Ivo

File: Language/bg/Cookie.php [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+ *
+ * (c) CodeIgniter Foundation <admin@codeigniter.com>
+ *
+ * For the full copyright and license information, please view
+ * the LICENSE file that was distributed with this source code.
+ */
+
+// Cookie language settings
+return [
+    'invalidExpiresTime'    => 'Invalid "{0}" type for "Expires" attribute. Expected: string, integer, DateTimeInterface object.',
```
Comment (paulbalandan) [inline Language/bg/Cookie.php:14]: This group should be removed.

File: README.md [hunk 2]
```diff
@@ -33,6 +33,7 @@ claims no expertise beyond English.
 | Locale | Language            | As of** | Github User           | Maintainer
 |--------|---------------------|---------|-----------------------| --------------------
 | ar     | Arabic              | 4.0.0   | [daif][ar]            | Daif Alotaibi
+| bg     | Bulgarian           | 4.0.0   | [ivcho02][bg]         | Ivaylo Tsvetanov
```
Comment (paulbalandan) [inline README.md:36]: Please change the version to 4.x as this is not yet released.

**Review by paulbalandan — COMMENTED** [review]
@ivcho02 Thanks for the changes.

There are styling comments flagged by the linter. Kindly fix those. And some comments from me.

**Review by paulbalandan — COMMENTED** [review]
There are still styling changes.
You can run `vendor/bin/php-cs-fixer fix -v` in your terminal to fix them all at once.

**Comment by ivcho02** [comment]
Thanks for the note, all files should be fixed now. :-)

File: Language/bg/Images.php [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    'rotateUnsupported'      => 'Изглежда ротацията на изображението не се поддържа от вашия сървър.',
+    'libPathInvalid'         => 'Пътят до вашата библиотека с изображения не е правилен. Моля, задайте правилния път в предпочитанията си за изображение. {0}',
+    'imageProcessFailed'     => 'Неуспешна обработка на изображението. Моля, проверете дали вашият сървър поддържа избрания протокол и че пътят до вашата библиотека с изображения е правилен.',
+    'rotationAngleRequired'  => 'Необходим е ъгъл на завъртане за завъртане на изображението.',
+    'invalidPath'            => 'Пътят до изображението не е правилен.',
+    'copyFailed'             => 'Програмата за копиране на изображението е неуспешна.',
+    'missingFont'            => 'Не може да се намери шрифт за използване.',
+    'saveFailed'             => 'Не може да се запази изображението. Моля, уверете се, че изображението и файловата директория могат да се записват.',
+    'invalidDirection'       => 'Посоката на обръщане може да бъде само `вертикална` или `хоризонтална`. Дадено: {0}',
+    'exifNotSupported'       => "Четенето на EXIF \u{200b}\u{200b}данни не се поддържа от тази PHP инсталация.",
```
Comment (paulbalandan) [inline Language/bg/Images.php:33]: Please remove the zero width space, then re-run `vendor/bin/php-cs-fixer fix -v`

File: README.md [hunk 4]
```diff
@@ -33,6 +33,7 @@ claims no expertise beyond English.
 | Locale | Language            | As of** | Github User           | Maintainer
 |--------|---------------------|---------|-----------------------| --------------------
 | ar     | Arabic              | 4.0.0   | [daif][ar]            | Daif Alotaibi
+| bg     | Bulgarian           | 4.x     | [ivcho02][bg]         | Ivaylo Tsvetanov
```
Comment (paulbalandan) [inline README.md:36]: I think you forgot to add your Github link below.

File: Language/bg/Images.php (hunk 3 above)
Comment (ivcho02) [reply Language/bg/Images.php]: Good catch, fixed.

**Review by paulbalandan — APPROVED** [review]

**Comment by paulbalandan** [comment]
Thank you, @ivcho02

**Pull request merged**